src/tests/functional/plugin/gpu/single_layer_tests/dynamic/top_k.cpp

```diff
@@ -19,16 +19,16 @@ using namespace ov::test;
 namespace GPULayerTestsDefinitions {
 
 typedef std::tuple<
-        int64_t,                           // keepK
-        int64_t,                           // axis
-        ngraph::opset4::TopK::Mode,        // mode
-        ngraph::opset4::TopK::SortType,    // sort
-        ElementType,                       // Net precision
-        ElementType,                       // Input precision
-        ElementType,                       // Output precision
-        InputShape,                        // inputShape
-        TargetDevice,                      // Device name
-        std::map<std::string, std::string> // Additional network configuration
+        int64_t,                            // keepK
+        int64_t,                            // axis
+        ngraph::opset4::TopK::Mode,         // mode
+        ngraph::opset4::TopK::SortType,     // sort
+        ElementType,                        // Net precision
+        ElementType,                        // Input precision
+        ElementType,                        // Output precision
+        InputShape,                         // inputShape
+        TargetDevice,                       // Device name
+        bool                                // Input style (true:constant, false:parameter)
```

Comment:
nit: you can use `ngraph::helpers::InputLayerType` to distinguish layer type for code readability

Reply:
done